Projects/Contracts/Sources/View/ContractTable.swift

```diff
@@ -35,39 +37,16 @@ struct ContractTable: View {
 
     var body: some View {
         VStack(spacing: 0) {
-            LoadingStoreViewWithContent(ContractStore.self, [.fetchContracts], [.fetchContracts], showLoading: false) {
-                hSection {
-                    PresentableStoreLens(
-                        ContractStore.self,
-                        getter: { state in
-                            getContractsToShow(for: state)
-
-                        }
-                    ) { contracts in
-                        VStack(spacing: .padding8) {
-                            ForEach(contracts, id: \.id) { contract in
-                                ContractRow(
-                                    image: contract.pillowType?.bgImage,
-                                    terminationMessage: contract.terminationMessage,
-                                    contractDisplayName: contract.currentAgreement?.productVariant.displayName ?? "",
-                                    contractExposureName: contract.exposureDisplayName,
-                                    activeFrom: contract.upcomingChangedAgreement?.activeFrom,
-                                    activeInFuture: contract.activeInFuture,
-                                    masterInceptionDate: contract.masterInceptionDate,
-                                    tierDisplayName: contract.currentAgreement?.productVariant.displayNameTier,
-                                    onClick: {
-                                        router.push(contract)
-                                    }
-                                )
-                                .fixedSize(horizontal: false, vertical: true)
-                                .transition(.slide)
-                            }
-                        }
-                    }
-                }
-                .presentableStoreLensAnimation(.spring())
-                .sectionContainerStyle(.transparent)
-            }
+            successView.loading($vm.viewState)
```

Comment:
We don't need both: 
loading
loadingWithButtonLoading
just loadingWithButtonLoading here